guardrails-scripts/yb-voyager-pg-grant-migration-permissions.sql

```diff
@@ -153,6 +153,19 @@ GRANT pg_read_all_stats to :voyager_user;
     \echo '--- Creating Replication Group ---'
     CREATE ROLE :replication_group;
 
+    -- Prompt the user to let them know that ownership of the tables will be transferred to the replication group and proceed only if they confirm
+    \echo ''
+    \echo 'This script will transfer ownership of all tables in the specified schemas to the specified replication group. The migration user and the original owner of the tables will be added to the replication group.'
+    -- Only 'yes' or 'no' are valid inputs. 'y' and 'n' are not valid inputs.
```

Comment:
nit: y/n might be simpler, so maybe choose that instead of yes/no

Reply:
Hey so the psql script does not support string comparision in the \if statement. It only supports a boolean variables over there i.e. 0 /1 or true/false or yes/no - all of these are considered boolean values. So we can only use yes/no